File: Backend/Entities/ExchangeCoordinator.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Backend.Entities
{
    /// <summary>A class for representing an exchange coordinator.</summary>
    [Table("ExchangeCoordinators")]
    public class ExchangeCoordinator : DomainUser
    {
        public DepartmentInfo Department { get; set; }

        public ICollection<ToDoItem> ToDoList { get; set; }
    }
}

File: Frontend/src/app/_models/exchange-coordinator.ts
import { DepartmentInfo } from './department-info';
import { ToDoItem } from './to-do-item';
import { DomainUser } from './domain-user';

export class ExchangeCoordinator extends DomainUser {
  department: DepartmentInfo;
  toDoList: ArrayLike<ToDoItem>;
}
